assume/markets/base_market.py

```diff
@@ -175,6 +175,9 @@ def handle_orderbook(self, content: dict, meta: dict):
                 ), f"minimum_bid_price {order['price']}"
 
                 if "bid_type" in order.keys():
+                    order["bid_type"] = (
+                        "SB" if order["bid_type"] == None else order["bid_type"]
+                    )
```

Comment:
As bid_type is not a default param, I think this would not belong here. But this will be fixed when we switch to a market_mechanism class. So its fine